seahorn/jobs/byte_cursor_read_u8/aws_byte_cursor_read_u8_harness.c

```diff
@@ -1,35 +1,27 @@
 /*
- * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
  *
- * Licensed under the Apache License, Version 2.0 (the "License").
- * You may not use this file except in compliance with the License.
- * A copy of the License is located at
- *
- *  http://aws.amazon.com/apache2.0
- *
- * or in the "license" file accompanying this file. This file is distributed
- * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
- * express or implied. See the License for the specific language governing
- * permissions and limitations under the License.
  */
 
+#include <seahorn/seahorn.h>
 #include <aws/common/byte_buf.h>
-#include <proof_helpers/make_common_data_structures.h>
-#include <proof_helpers/proof_allocators.h>
+#include <proof_allocators.h>
+#include <byte_buf_helper.h>
+#include <utils.h>
 
-void aws_byte_cursor_read_u8_harness() {
+int main() {
     /* parameters */
     struct aws_byte_cursor cur;
+    initialize_byte_cursor(&cur);
     size_t length;
-    __CPROVER_assume(length >= 1);
+    assume(length >= 1);
```

Comment:
undefined behavior. `length` is not initialized.

Check all other assumptions for undefined behavior. I might have missed other cases.